src/components/standalone/monitoring/NetifyInformaticsContent.vue

```diff
@@ -0,0 +1,188 @@
+<!--
+  Copyright (C) 2024 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { ref, onMounted, watch } from 'vue'
+import type { Ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+import FormLayout from '@/components/standalone/FormLayout.vue'
+import { useNotificationsStore } from '@/stores/notifications'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import { NeButton } from '@nethesis/vue-components'
+import {
+  NeSkeleton,
+  NeFormItemLabel,
+  NeTextInput,
+  getAxiosErrorMessage,
+  NeInlineNotification,
+  NeBadge
+} from '@nethesis/vue-components'
+import { NeToggle } from '@nethesis/vue-components'
+import { ubusCall } from '@/lib/standalone/ubus'
+import { AxiosError } from 'axios'
+
+const notificationsStore = useNotificationsStore()
+const { t } = useI18n()
+
+interface Form {
+  status: boolean
+  uuid: string
+}
+
+const form = ref<Form>({
+  status: false,
+  uuid: ''
+})
+
+interface ErrorObject {
+  notificationTitle: string
+  notificationDescription: string
+  uuid: string
+}
+const objError: ErrorObject = {
+  notificationTitle: '',
+  notificationDescription: '',
+  uuid: ''
+}
+
+let isError = ref(false)
+let loading = ref(false)
+let saving = ref(false)
+let uuidRef = ref()
+let JustCopied = ref(false)
+
+const error: Ref<ErrorObject> = ref({ ...objError })
+const errorLoadingConfiguration: Ref<ErrorObject> = ref({ ...objError })
+const errorSaving: Ref<ErrorObject> = ref({ ...objError })
+
+onMounted(() => {
+  getConfiguration()
+})
+
+async function getConfiguration() {
+  try {
+    loading.value = true
+    let getDataConfiguration = await ubusCall('ns.netifyd', 'status', {})
+    if (getDataConfiguration && getDataConfiguration.data) {
+      let configuration = getDataConfiguration.data
+      form.value.status = configuration.enabled
+      form.value.uuid = configuration.uuid
+      loading.value = false
```

Comment:
Put this into a "finally" clause, this way the loading always removes itself after the request